src/doc/palette.cpp

```diff
@@ -400,6 +408,117 @@ int Palette::findBestfit2(int r, int g, int b, int a) const
   return bestfit;
 }
 
+// Auxiliary function for rgbToOtherSpace()
+double f(double t)
+{
+  if (t > 0.00885645171)
+    return pow(t, 0.3333333333333333);
```

Comment:
Use `std::` namespace for standard functions. And include `<cmath>`:
```suggestion
    return std::pow(t, 0.3333333333333333);
```